types/micromodal/index.d.ts

```diff
@@ -8,10 +8,10 @@
  */
 export interface MicroModalConfig {
     /** This is fired when the modal is opening. */
-    onShow?: ((modal?: HTMLElement) => void) | undefined;
+    onShow?: ((modal?: HTMLElement, trigger?: HTMLElement, event?: Event) => void) | undefined;
```

Comment:
I think the `event` parameters can also be `null` (happens if the modal is shown/closed programmatically):

```suggestion
    onShow?: ((modal?: HTMLElement, trigger?: HTMLElement, event?: Event | null) => void) | undefined;
```